src/components/cap-reporter.js

```diff
@@ -22,26 +96,31 @@ export default class CapReporter extends LitElement {
 
 	render() {
 		return html`
-			<h1>${this.reporterData.short_name}</h1>
-			<h2>
-				${this.reporterData.full_name}
-				(${this.reporterData.start_year}-${this.reporterData.end_year})
-			</h2>
-			Volume number:
-			<ul>
-				${this.volumesData
-					.sort((a, b) => a.volume_number - b.volume_number)
-					.map(
-						(v) =>
-							html`<li>
-								<a
-									href="/caselaw/?reporter=${this
-										.reporter}&volume=${v.volume_number}"
-									>${v.volume_number}</a
-								>
-							</li>`,
-					)}
-			</ul>
+			<cap-caselaw-layout>
+				<div class="reporters__main">
+					<h1 class="reporter__heading">${this.reporterData.short_name}</h1>
+					<h2 class="reporter__subHeading">
+						${this.reporterData.full_name}
+						(${this.reporterData.start_year}-${this.reporterData.end_year})
+					</h2>
```

Comment:
This wasn't introduced in your PR, so I apologize for not spotting it sooner. According to a 2022 blog post by the accessibility consultant group TPGi, using a heading element to markup an alternative title is well-intentioned but not recommended.  

They recommend using a `p` tag instead, in conjunction with the `hgroup` element, for example: 

```
<hgroup>
<h1>All At Sea Over Ventriloquism<h1> 
<p>Captain Dick's Logbook</p> 
<p>By Richard P. Wightman alias Captain Dick</p>
</hgroup>
```

It's not entirely clear to me if `reporterData.short_name` is the alternative title, and the `full_name` is the formal title — I think if "Ark." isn't the title that would be cited in context, we can make that a `p` element and the other title an `h1.` 

The blog post I learned this from is [Subheadings, subtitles, alternative titles and taglines in HTML](https://www.tpgi.com/subheadings-subtitles-alternative-titles-and-taglines-in-html/)

Reply:
One addendum: according to Jack, "Ark. is the bluebook way of referring to the Aransas Reports in citations inserted into cases"

If that's the case, I think the short name could be the `h1` and the full, descriptive name could be the `p` — does that make sense?